app/src/main/java/org/p2p/wallet/home/ui/main/adapter/TokenAdapter.kt

```diff
@@ -17,7 +17,7 @@ class TokenAdapter(
 
     fun setItems(new: List<HomeElementItem>, isZerosHidden: Boolean, state: VisibilityState) {
         this.isZerosHidden = isZerosHidden
-        val old = ArrayList(data)
+        val old = data.toMutableList()
```

Comment:
`ArrayList(data)` - is a way of deep copying all the elements and creating a new reference. 
I'm not sure that `.toMutableList()` can achieve the same.

In my opinion, when you call `data.clear()` - `old` will clear too.

Reply:
???
![image](https://user-images.githubusercontent.com/99185280/160404763-5ddb72ce-b267-498c-a418-25076e094450.png)

Reply:
i presume it is same as ArrayList(data)  ? isn't it ?

Reply:
damn, yes it is. sorry

Reply:
damn, yes it is. sorry